pic32/variants/Uno32/Board_Defs.h

```diff
@@ -155,6 +155,9 @@
 #define	PIN_INT3	8
 #define	PIN_INT4	35
 
+#define NOT_AN_INTERRUPT -1
+#define digitalPinToInterrupt(p) ((p) == PIN_INT0 ? 0 : ((p) == PIN_INT1 ? 1 : ((p) == PIN_INT2 ? 2 : ((p) == PIN_INT3 ? 3 : ((p) == PIN_INT4 ? 4 : NOT_AN_INTERRUPT)))))
```

Comment:
I wouod suggest moving this to a separate PR so it  an be built up and tested properly including the default fallback and any special _PGM tables that may be required for othe rboatds. There are many boards that need to have it implemented and tested, and some may need special care with PPS.